src/routes/test/[slug]/page.test.ts

```
@@ -0,0 +1,200 @@
+import * as getCandidateHelper from '$lib/helpers/getCandidate';
+import * as testServer from '$lib/server/test';
+import { fail, redirect } from '@sveltejs/kit';
```

ESLint

[error] 'redirect' is defined but never used.

(@typescript-eslint/no-unused-vars)